package.json

```diff
@@ -14,7 +14,7 @@
     "test": "make test"
   },
   "dependencies": {
-    "debug": "~2.6.4",
+    "debug": "~2.6.8",
     "notepack.io": "~2.1.0",
     "redis": "2.6.3",
     "socket.io-parser": "2.3.2"
```

Comment:
This dep ALSO needs updating. (this is where the vulnerability is being reported (by the service I use)

Reply:
`socket.io-parser` would need to be upgraded to at the very least `~3.1.0` in order to clear up the hard lock on `debug@2.3.3`.

I have a feeling if this is updated to that Major release, the vunerabilies will clear up.
@shian15810 you can make the change on this PR or I could submit another PR instead.

totes up to you.